torchmetrics/image/ssim.py

```diff
@@ -185,18 +201,24 @@ def __init__(
         self.add_state("preds", default=[], dist_reduce_fx="cat")
         self.add_state("target", default=[], dist_reduce_fx="cat")
 
-        all_kernel_ints = all(isinstance(ks, int) for ks in kernel_size)
-        if not isinstance(kernel_size, Sequence) or len(kernel_size) != 2 or not all_kernel_ints:
+        if not (isinstance(kernel_size, Sequence) or isinstance(kernel_size, int)):
             raise ValueError(
-                "Argument `kernel_size` expected to be an sequence of size 2 where each element is an int"
-                f" but got {kernel_size}"
+                "Argument `kernel_size` expected to be an sequence or an int." f"or a single int. Got {kernel_size}"
             )
-        self.kernel_size = kernel_size
+        if isinstance(kernel_size, Sequence) and (
+            len(kernel_size) not in (2, 3) or not all(isinstance(ks, int) for ks in kernel_size)
+        ):
+            raise ValueError(
+                "Argument `kernel_size` expected to be an sequence of size 2 or 3 where each element is an int, "
+                f"or a single int. Got {kernel_size}"
+            )
```

Comment:
please add a `Raises` section to the docstring similar to:
https://github.com/PyTorchLightning/metrics/blob/5cbc7add8877e6524b140eb35f57a26b53fe5670/torchmetrics/image/fid.py#L182-L188